### ISSUE TITLE
Configure path to store oedt databases

### ISSUE THREAD

**Issue by lievencardoen** [omecra/de.omecra.oedt.updatesite#53]
Is there a way to store the OEDT databases on another drive for example? To configure the path?

**Comment by lneumeier** [comment]
No, this is currently not possible. Have you tried to create a junction for the folder de.omecra.oedt.core ?

**Comment by lievencardoen** [comment]
I'll have a try! thx.
